test/hack/cleanup/main.go

```diff
@@ -208,3 +251,23 @@ func getOldLaunchTemplates(ctx context.Context, ec2Client *ec2.Client, expiratio
 	}
 	return names
 }
+
+func getOldOICD(ctx context.Context, iamClient *iam.Client, expirationTime time.Time) (names []string) {
+	testSuite := []string{"upgrade", "chaos", "consolidation", "drift", "integration", "interruption", "ipv6", "machine", "scale", "utilization"}
```

Comment:
Is it possible for us to identify this OIDC provider by some other tag that is only added by a test run? Otherwise, we are going to have to keep expanding this list as we add tests and we are going to forget to maintain this list

Reply:
Any chance we can discover these names by introspecting the `.github/workflows` file or by reading the directory names. I have some concern that if we specify the test suites like this we are going to continually forget to update these values.

Reply:
Another option is to do the inverse i.e. any OIDC provider that doens't have the `do-not-delete` tag gets cleaned up.

Reply:
We don't create the OIDCs, so I'm not sure how we can tag them